cumulus/pallets/parachain-system/src/validate_block/trie_cache.rs

```diff
@@ -29,7 +29,7 @@ use trie_db::{node::NodeOwned, Hasher};
 /// have already been loaded and decoded from the storage proof.
 pub(crate) struct TrieCache<'a, H: Hasher> {
 	node_cache: RefMut<'a, BTreeMap<H::Out, NodeOwned<H::Out>>>,
-	value_cache: Option<RefMut<'a, BTreeMap<Box<[u8]>, trie_db::CachedValue<H::Out>>>>,
+	value_cache: Option<RefMut<'a, &'a mut BTreeMap<Box<[u8]>, trie_db::CachedValue<H::Out>>>>,
```

Comment:
```suggestion
	value_cache: Option<RefMut<'a, BTreeMap<Box<[u8]>, trie_db::CachedValue<H::Out>>>>,
```
This needs to go for it to compile.

Reply:
It compiled locally for me 🙃